### PR TITLE
Remove references to tornasole

### PR DESCRIPTION
**Pull request by jarednielsen**
### Description of changes:

The only remaining reference is the environment variable `TORNASOLE_EVENT_FILE_RETRY_LIMIT`, addressed in PR #91 .

The S3 "state" that we're currently relying on is:
```
s3://smdebug-testing/resources/event-files-missing
s3://smdebug-testing/resources/event-files-missing-incomplete
s3://smdebug-testing/resources/has_step_scenarios
s3://smdebug-testing/resources/one-index-file
s3://smdebug-testing/resources/dist-logs-10
s3://smdebug-testing/resources/collection-tests
```

I'll create an issue to move away from relying on resources.

#### Style and formatting:

I have run `pre-commit install` to ensure that auto-formatting happens with every commit.

#### Issue number, if available

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: examples/mxnet/notebooks/mxnet-tensor-plot.ipynb [hunk 1]
```diff
@@ -78,7 +78,7 @@
     "REGION='us-west-2'\n",
     "TAG='latest'\n",
     "\n",
-    "docker_image_name= '072677473360.dkr.ecr.{}.amazonaws.com/tornasole-preprod-mxnet-1.4.1-cpu:{}'.format(REGION, TAG)\n",
+    "docker_image_name= '072677473360.dkr.ecr.{}.amazonaws.com/SageMaker Debugger-preprod-mxnet-1.4.1-cpu:{}'.format(REGION, TAG)\n",
```
Comment (Vikas-kum) [inline examples/mxnet/notebooks/mxnet-tensor-plot.ipynb]: docker image name with space?

File: docs/api.md [hunk 2]
```diff
@@ -310,7 +310,7 @@ Sample JSON file:
 In SageMaker environment, the presence of this JSON is necessary to log any Tensorboard artifact.
 By default, this path is set to point to a pre-defined location in SageMaker.
 
-tensorboard_dir can also be passed while creating the hook [Creating a hook](###Hook from Python) using the API or
+tensorboard_dir can also be passed while creating the hook Creating a hook using the API or
```
Comment (rahul003) [inline docs/api.md]: Something's off here

File: tests/analysis/trials/test_has_passed_step_scenarios.py [hunk 3]
```diff
@@ -20,7 +20,7 @@ def test_single_writer_all_steps_written_complete_job():
     END_OF_JOB.ts --> Present
     """
 
-    path = "s3://tornasole-testing/has_step_scenarios/single-writer-all-steps-written-complete-job"
+    path = "s3://smdebug-testing/has_step_scenarios/single-writer-all-steps-written-complete-job"
```
Comment (rahul003) [inline tests/analysis/trials/test_has_passed_step_scenarios.py]: I believe these paths have some files needed for the tests. Have you copied them over to new location?

Comment (rahul003) [reply tests/analysis/trials/test_has_passed_step_scenarios.py]: If you have not already, can you put them in a resources directory, so we know that we can clean everything except resources

File: tests/analysis/trials/test_has_passed_step_scenarios.py [hunk 4]
```diff
@@ -458,7 +454,7 @@ def test_override_if_too_many_steps_skipped():
 
     os.environ["INCOMPLETE_STEP_WAIT_WINDOW"] = "10"
 
-    path = "s3://tornasole-testing/has_step_scenarios/too-many-steps-skipped"
+    path = "s3://smdebug-testing/has_step_scenarios/too-many-steps-skipped"
```
Comment (rahul003) [inline tests/analysis/trials/test_has_passed_step_scenarios.py]: There are many tests where my above comment is applicable. Check with @NihalHarish

File: tests/analysis/trials/test_has_passed_step_scenarios.py (hunk 3 above)
Comment (jarednielsen) [reply tests/analysis/trials/test_has_passed_step_scenarios.py]: Done. We should really move away from relying on S3 state for tests, spoke with Nihal about it.

File: tests/analysis/trials/test_has_passed_step_scenarios.py (hunk 4 above)
Comment (jarednielsen) [reply tests/analysis/trials/test_has_passed_step_scenarios.py]: Done

File: smdebug/trials/trial_catalog.py [hunk 5]
```diff
@@ -34,7 +34,7 @@ def __init__(self,endpoint,port):
         self.endpoint = endpoint
         self.port = port
         self.client = InfluxDBClient(host=self.endpoint, port=self.port)
-        self.client.switch_database('tornasole_deb')
+        self.client.switch_database('deb')
```
Comment (Vikas-kum) [inline smdebug/trials/trial_catalog.py]: is this class needed? not in this PR but subsequent PR we should remove this class. can you track it in github issue

File: tests/analysis/tensors/test_refresh.py [hunk 6]
```diff
@@ -13,12 +13,10 @@
 @pytest.mark.slow  # 0:38 to run
 def test_refresh_tensors():
     trial_name = str(uuid.uuid4())
-    path = "s3://tornasole-testing/rules/tensors/ts_output/train/"
+    path = f"s3://smdebug-testing/outputs/rules_refresh_tensors-{uuid.uuid4()}/"
```
Comment (Vikas-kum) [inline tests/analysis/tensors/test_refresh.py]: all s3 tests should not be part of unit test run. is slow already doing this?

File: tests/analysis/tensors/test_refresh.py [hunk 7]
```diff
@@ -31,21 +29,15 @@ def test_refresh_tensors():
     tr = create_trial(path + trial_name)
     assert len(tr.steps()) == 4
 
-    try:
+    with pytest.raises(TensorUnavailable):
```
Comment (Vikas-kum) [inline tests/analysis/tensors/test_refresh.py]: neat

File: tests/analysis/tensors/test_refresh.py (hunk 6 above)
Comment (jarednielsen) [reply tests/analysis/tensors/test_refresh.py]: Running `pytest tests --skipslow` will run a subset of the tests as a sanity check before passing CI. S3 tests are currently a part of CI.

**Comment by Vikas-kum** [comment]
Nice!

**Pull request merged**